make_wheels.py

```diff
@@ -13,15 +13,15 @@
 ZIG_VERSION_INFO_URL = 'https://ziglang.org/download/index.json'
 ZIG_PYTHON_PLATFORMS = {
     'x86_64-windows': 'win_amd64',
+    'x86-windows':   'win32',
```

Comment:
I think you're missing a space.

```suggestion
    'x86-windows':    'win32',
```